docs/README.md

```diff
@@ -4,13 +4,167 @@
 
 ### An ethernet faking WiFi and LoRa host for data transfer without signal.
 
-Magic WiFi is a service built by students interested in farming data in limited connection areas and looking for a low cost method
-to collect data without satellites.
-It also lines up well to work in racing events, such as Baja racing where crews are limited but may need data from the vehicle while in operation so that
-maintenance can be planned for next resupply.
+Magic Wifi is a system that creates a flexible and dependable wireless connection between two devices at variable ranges. All the technology behind Magic Wifi is hidden from the two user devices which believe they are simply connected with an Ethernet cable at both ends. Underneath, Magic Wifi employs both LoRa and Wifi technologies to create a dynamic connection that seamlessly switches between long-range, low-bandwidth rates for critical data like telemetry and short-range, high-bandwidth rates for large data dumps. 
+
+Magic Wifi can be used in various different settings where two devices need to be connected at different distances. One example application is endurance racing, like Le Mans. Critical data is needed at all times to inform the pit crew of components that require maintenance at the next pit stop. High volume data, by contrast, would include detailed engine diagnostics or telemetry logs. The detailed data would be helpful to the technicians during pit stops to complete repairs and tune the cars for future laps but is not as critical.  Racing is not the only important application. For farms in remote areas, cellular coverage can often be poor to non-existent, making internet connectivity scarce outside of the farmer’s home. Magic Wifi provides a low-speed connection over multiple miles so that the farmer can communicate with others inside the house. The close-range connectivity allows the farmer to transfer large volumes of important statistics and pictures of crops without the hassle of removing internal storage cards or connecting Ethernet cables. These two are just a few examples of where Magic Wifi would be useful. 
+
+The final Magic Wifi product can successfully send Ethernet frames over both Wifi and LoRa. However, it remains very susceptible to interference, both on Wifi and LoRa which led to our live demo not working properly at the expo. Even at our later instructor demo, this remained an issue in campus buildings so we had to show a recording of certain features.  At the beginning we had concerns about sending frames over LoRa due to the low bandwidth and need for subframing due to a small maximum LoRa frame size.

```

Comment:
Lots of stuff here that makes sense in the context of where it was copied from but probably doesn't belong in this repo :)
(Interference bits certainly relevant! Maybe just not the context of where we encountered them, etc)

Reply:
Was copy pasting a lot and must have graabed a little extra on accident, good catch